neural_speed/models/chatglm/chatglm.h

```diff
@@ -26,7 +26,7 @@ enum chatglm_model {
 static const model_scratch chatglm_mem_req(int n_layers) {
   switch (n_layers) {
     case 28:
-      return {2048ull * MB, 2048ull * MB, 4096ull * MB};
+      return {4096ull * MB, 4096ull * MB, 8192ull * MB};
```

Comment:
Does it still need the larger memory when `MAX_REQUEST_NUM=1`?

Reply:
Chatglm2 claims that it supports 32k tokens as input. But a little less memory here is no problem.